### ISSUE TITLE
Feature request: alarm status

### ISSUE THREAD

**Issue by andreabafile** [audiconnect/audi_connect_ha#378]
Is there any chance to add a sensor for the alarm status and an additional one for the alarm trigger reasons?

Thank you.

**Comment by coreywillwhat** [comment]
Hi @andreabafile,
Thanks for the suggestion! 

I've just tried adding `theftAlarm` to the JOBS2QUERY (this shows as an available scope to query) but no additional data is returned. This is likely an outdated part of the API. Note I've tried a lot of the other scopes as well and I haven't seen anything else pull in yet.

Have you tried triggering your alarm and seeing if anything changes in the Audi app to show the alarm is triggered and the reason? If it shows up in the app, there is a strong chance we could find it. Maybe it shows up under warningLights. 

Otherwise it doesn't look like there is anything available on the backend, at least while the alarm isn't triggered.

Give it a test and let me know about the Audi App.
